src/app/components/GearCardConnection/GearCardConnection.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Cable_1 from '../assets/cables/Cable1_Icon';
+import Cable_2 from '../assets/cables/Cable2_Icon';
+import Cable_3 from '../assets/cables/Cable3_Icon';
+import Cable_4 from '../assets/cables/Cable4_Icon';
+import Cable_5 from '../assets/cables/Cable5_Icon';
+import Cable_6 from '../assets/cables/Cable6_Icon';
+import Cable_7 from '../assets/cables/Cable7_Icon';
+import Typography from '../Typography/Typography';
+import style from './GearCardConnection.module.css';
+
+type GearConnectionProps = {
```

Comment:
This doesn't match the filename or the component name

Reply:
Resolved in [76b9e9b](https://github.com/Seborider/plugs./pull/33/commits/76b9e9b7203ba3524422a0f2dfbae26dfe148aa8)